cylc/flow/cfgspec/workflow.py

```diff
@@ -260,7 +256,7 @@ def get_script_common_text(this: str, example: Optional[str] = None):
             time zones will default to UTC (``Z``) unless
             :cylc:conf:`flow.cylc[scheduler]cycle point time zone` or
             is set or
-            :ref:`Cylc 7 back compatibility mode is enabled <backcompat>`.
+            :ref:` <Cylc_7_compat_mode>` is enabled.
```

Comment:
I think this should either be
```suggestion
            :ref:`Cylc 7 back compatibility mode <Cylc_7_compat_mode>` is enabled.
```
or
```suggestion
            :ref:`Cylc_7_compat_mode` is enabled.
```